packages/fxa-settings/package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": "https://accounts.firefox.com/beta/settings",
   "private": true,
   "scripts": {
-    "build-postcss": "postcss src/styles/tailwind.css -o src/styles/tailwind.out.css",
+    "build-postcss": "postcss src/styles/tailwind.scss -o src/styles/tailwind.out.scss",
```

Comment:
"Why is this outputting SCSS here but our other Tailwind setups output CSS?" Because we don't import SCSS in those, and CRA/Storybook/Tailwind setups don't care what they receive now. I am only changing this because it itself because it imports SCSS files.

Reply:
I should note that at any point if we _do_ start importing SCSS into those other files, it's as simple as changing the extension.